catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -356,6 +356,14 @@
         </bundle>
     </feature>
 
+    <feature name="catalog-core-validator" install="manual" version="${project.version}"
```

Comment:
Is there any reason we shouldn't make this installed by default and ship with a "basic" validation definition file?